tests/federation/transport/test_client.py

```diff
@@ -94,3 +95,34 @@ def test_servers_in_room(self) -> None:
         # Retrieve and check the parsed SendJoinResponse
         parsed_response = parser.finish()
         self.assertEqual(parsed_response.servers_in_room, ["hs1", "hs2"])
+
+    def test_errors_closing_coroutines(self) -> None:
```

Comment:
docstring please, just so the context is kept around for years to come

Reply:
Sure, this is 99a0ad8.